<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_camera_shutter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>720</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-image: url(:/images/img//Aperture.jpg);
border: 1px solid #2E7D32;
border-radius: 5px; 
padding: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_background">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>393</width>
     <height>852</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FAFAFA;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_home">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>840</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-image: url(:/images/img//home.png);
border: 1px solid #2E7D32;
border-radius: 5px; 
padding: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_profile">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>840</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-image: url(:/images/img//User.jpg);
border: 1px solid #2E7D32;
	image: url(:/images/src/User.jpg);
border-radius: 5px; 
padding: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_camera">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>830</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-image: url(:/images/img//VideoStabilization.png);
border: 1px solid #2E7D32;
border-radius: 5px; 
padding: 5px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_menubar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>821</y>
     <width>393</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 79, 254);
border-top-right-radius: 14px;
border-top-left-radius: 14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_webcam">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>393</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/img//try-this-trainer-approved-chest-day-workout.jpg);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_shutter_effect">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>393</width>
     <height>781</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_file">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>760</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/img/Add_Image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_camera_box">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>754</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lb_background</zorder>
  <zorder>lb_webcam</zorder>
  <zorder>lb_shutter_effect</zorder>
  <zorder>lb_menubar</zorder>
  <zorder>btn_home</zorder>
  <zorder>btn_profile</zorder>
  <zorder>btn_camera</zorder>
  <zorder>btn_camera_shutter</zorder>
  <zorder>lb_camera_box</zorder>
  <zorder>btn_file</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
